cypress.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
more of a question, i'm guessing this is needed even if empty? is this where we configure cypress?

Reply:
I think its still needed even inf empty, but in future will put the localhost address in there actually so that we can run locally outside docker